labours.py

```diff
@@ -419,8 +419,9 @@ def train_embeddings(coocc_tree, tmpdir, shard_size=4096):
     from scipy.sparse import csr_matrix
     try:
         from . import swivel
-    except SystemError:
+    except ImportError:
```

Comment:
```
In [4]: from . import test_artm_model
---------------------------------------------------------------------------
SystemError                               Traceback (most recent call last)
<ipython-input-4-e6adabd80afa> in <module>()
----> 1 from . import test_artm_model

SystemError: Parent module '' not loaded, cannot perform relative import
```

This was changed in 3.6: https://github.com/python/cpython/blob/8a9cd20edca7d01b68292036029ae3735ce65edd/Misc/NEWS.d/next/Core%20and%20Builtins/2017-07-11-06-31-32.bpo-30876.x35jZX.rst